src/postman/operation.ts

```diff
@@ -26,18 +32,37 @@ export const transformPostmanCollectionOperation: PostmanCollectionHttpOperation
     throw new Error(`Unable to find "${method} ${path}"`);
   }
 
+  const auth = item.getAuth();
+  const postmanSecurity = auth && findPostmanSecurityScheme(auth, collection);
+  const request = transformRequest(item.request);
+  const security: HttpSecurityScheme[][] = [];
+
+  if (postmanSecurity) {
+    switch (postmanSecurity.type) {
+      case 'securityScheme':
+        security.push([postmanSecurity.securityScheme]);
+        break;
+      case 'headerParams':
+        request.headers!.push(...postmanSecurity.headerParams);
+        break;
+      case 'queryParams':
+        request.query!.push(...postmanSecurity.queryParams);
+        break;
+    }
+  }
+
   return {
     id: '?http-operation-id?',
     iid: item.id,
     description: item.description && transformDescriptionDefinition(item.description),
     method,
     path,
     summary: item.name,
-    request: transformRequest(item.request),
+    request,
     responses: item.responses.map(transformResponse),
+    security,
     /*
     servers: ...,
-    security: ...,
```

Comment:
Good to see with every PR one of these weird pieces going away :)

Reply:
Yeah :)